doc/all.mk

```diff
@@ -49,15 +49,27 @@ install.doc: $(DOCINSTALL)
 clean.doc:
 	${Q}rm -f *~ rfc/*~ examples/*~ $(ADOC_FILES) $(HTML_FILES) $(PDF_FILES)
 
+
+#
+#  Check if the pandoc exist
+#
+PANDOC_EXISTS := $(shell which pandoc 2>/dev/null)
+ifeq "$(PANDOC_EXISTS)" ""
```

Comment:
Can't you just `ifeq "$(shell which pandoc 2>/dev/null)" ""`?

Reply:
but, yes. pretty good, I updated the pr.

I just copied/paste the `AUTOCONF_EXISTS` from Makefile. :)